### PR TITLE
add HelpViewer recipe

### PR DESCRIPTION
**Pull request by bbjimmy**
This is a pre-built yab app

### CODE REVIEW

File: haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe [hunk 1]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="A help file viewer"
+DESCRIPTION="An application for viewing help files that use a simple html like mark-up. Unlike html, .hlp is directed at help files not a rich web experiance."
```
Comment (Begasus) [inline haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:2]: Looks good so far, could you cut the DESCRIPTION lines to match our guidelines (80 char limit): https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description

Comment (bbjimmy) [reply haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe]: Done.

File: haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe [hunk 2]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="A help file viewer"
+DESCRIPTION="An application for viewing help files that use a simple html like mark-up. "
```
Comment (Begasus) [inline haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:2]: You could keep the full description and wrap the line with a backslash to move to a new line like:
```
DESCRIPTION="An application for viewing help files that use a simple html like \
mark-up.
Unlike html, .hlp is directed at help files not a rich web experiance."
```

Comment (Begasus) [reply haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:2]: Starting a new line without the backslash with move the new line also in HaikuDeport

File: haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	"
+REQUIRES="
+	haiku
+	yab >= 1.7.5.5
+	lib:libyab
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel >= r1~alpha4_pm_hrev49976
```
Comment (Begasus) [inline haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:28]: Shoudn't need versioning here, just "haiku_devel"

File: haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+INSTALL()
+{
+	mkdir -p $appsDir
+	mkdir $appsDir/HelpViewer
+	mkdir -p $dataDir
+	mkdir -p $binDir
+	cp -r apps/* $appsDir/HelpViewer/
+	chmod a+x $appsDir/HelpViewer/HelpViewr-post-install.sh
+	cp -r data/mime_db $dataDir/
+	ln -s $appsDir/HelpViewer/HelpViewer $binDir
```
Comment (Begasus) [inline haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:50]: Maybe add a Deskbar link here with addAppDeskbarSymlink

Comment (bbjimmy) [reply haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:50]: This is done intentionally. Launching the program by its self is only needed for developers. The prlgram should be launched from a control in the program that HelpViewer is used to document.

Comment (Begasus) [reply haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe:51]: Thanks for clearing this up :+1:

**Pull request merged**